posthog/src/main/java/com/posthog/internal/replay/RRAddedNode.kt

```diff
@@ -2,6 +2,7 @@ package com.posthog.internal.replay
 
 import com.posthog.PostHogInternal
 
+// Also used for UpdatedNode
 @PostHogInternal
 public class RRAddedNode(
```

Comment:
I went through the same journey in the transformer with `updates` being sent as additions.

Ended up renaming so that `adds` and `updates` are typed as `MobileNodeMutation` because it felt mismatched to need the same comment....

How about `RRMutatedNode` ??

Reply:
Yep, that sounds good.